package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "game",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development",
+    "serv": "webpack-dev-server --mode development",
+    "prod": "webpack --mode production",
+    "watch": "webpack --mode development --watch",
+    "style": "sass --watch src/assets/sass/style.scss src/style.css"
+  },
```

Comment:
I haven't thought that someone would run my project), also lint was configured a long time ago, when I started to configure webpack the first time, so I'm not sure what steps you need to take to get it working) What I can tell so far: there Eslint plugin on vscode, there's eslint config file and package.json. All I do is run npm install, and turn on Eslint plugin on the right-bottom corner of vscode. 
Anyway, I'll reconfigure lint on the last task, with kipping your suggestions in mind)

Reply:
Haha, you should always keep in mind that not only you will be working on the project you set up, so that anyone can work with you, especially at open-source projects. The better set up description, the easier to collaborate with each other :)  and please, don’t rely on vs code settings, they are usually personalized, that’s why we need to set up eslint and prettier plugins, so that we get rid of extra lines of formatted code submitted from different people